client/src/components/Applications/AppBrowser/AppBrowser.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import renderComponent from 'utils/testing';
+import AppBrowser from './AppBrowser';
+import { appTrayExpectedFixture } from "../../../redux/sagas/fixtures/apptray.fixture";
+
+const mockStore = configureStore();
+
+describe('App Browser', () => {
+  it('should render App Browser with different categories and app counts', () => {
+    const store = mockStore({
+      apps: appTrayExpectedFixture
+    });
+    const { getByText } = renderComponent(<AppBrowser />, store);
+
+    expect(getByText('Data Processing [3]')).toBeDefined();
```

Comment:
Maybe remove the numbers? just in case the fixtures have to change for any reason.

Reply:
I prefer checking for the numbers as it seems important that the numbers are there and correct.  If the fixtures change, we can just update the numbers.